core/src/avm1/value.rs

```diff
@@ -90,6 +90,12 @@ impl<'gc> From<u32> for Value<'gc> {
     }
 }
 
+impl<'gc> From<u64> for Value<'gc> {
+    fn from(value: u64) -> Self {
+        Value::Number(value as f64)
+    }
+}
+
```

Comment:
I get nervous about these kind of conversions, because `u64` -> `f64` can be lossy and `from::From` by convention should be a lossless conversion. But we already have `From<usize>`, so this is fine for now.